### DISCUSSION TITLE
Replace the cache with a background fetch

### DISCUSSION THREAD

**Discussion by tukwan** [arthurfiorette/axios-cache-interceptor#760] [Q&A] [answered]
Is it possible to return the current cache and still trigger the background API fetch to replace/update this cache ?

**Comment by arthurfiorette** [comment]
https://twitter.com/adamdehaven/status/1733228594313277608

**Reply by jdoklovic** [reply]
what's the answer? The twitter post doesn't exist now that it's X

**Reply by arthurfiorette** [reply, marked as answer]
<img width="625" height="927" alt="image" src="https://github.com/user-attachments/assets/2245eae6-027b-4997-9bb1-f7f6f5bd0d8f" />
